assets/sass/05-blocks/latest-posts/_editor.scss

```
@@ -15,11 +15,16 @@
 		}
 	}
 
-	&.is-grid > li {
-		margin-bottom: var(--global--spacing-vertical);
+	&.is-grid {
+		word-wrap: break-word;
+		word-break: break-word;
+	
+		& > li {
```

```suggestion

		> li {
```

This will fix the stylelint issues, there's some whitespace, and the & isn't necessary.

---

Thanks, I now have double checked everything with the linter!